### PR TITLE
es: Update HTML5_Gamedev_Phaser_Device_Orientation

### PR DESCRIPTION
**Pull request by dabl03**
I just fixed a couple of broken links. I plan on later doing other documentation for version 3 of phaser in the future (if I have time).

### Description

I got some dead links and decided to fix them.

### Motivation

Little thing, I only entered to learn a little about the mechanics of phaser (I haven't memorized the phaser 3 api yet) and I got some broken links and fixed them

### CODE REVIEW

File: files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md
Comment (Graywolf9) [inline files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md:20]: El enlace automáticamente dirige a la versión actual, también es el enlace utilizado en la versión en inglés, si crees que es mejor apuntar a una versión, sugerir el cambio en inglés y una vez aprobado hacer el cambio aquí

```suggestion
[Phaser](http://phaser.io/) es un _framework_ para construir juegos, de móvil o escritorio, en HTML5 . Es nuevo pero está creciendo velozmente gracias a la apasionada comunidad involucrada en el proceso de desarrollo. Puedes chequearlo [en GitHub](https://github.com/photonstorm/phaser) donde se encuentra como _open source._ Lee [la documentación](https://phaser.io/docs/) en línea y recorre su gran colección de [ejemplos](https://phaser.io/examples). El _framework_ Phaser provee una serie de herramientas que acelerarán el desarrollo y te ayudaran a manejar las tareas genéricas necesarias para completar tu juego, para que así puedas concentrarte en la idea del juego en sí.
```

**Review by Graywolf9 — CHANGES_REQUESTED** [review]
Muchas gracias! Dejo una pequeña observación

File: files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md
Comment (dabl03) [reply files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md:20]: It is better to target a specific version, because the tutorial is made for Phaser 2.2.2 which is somewhat different in structure than 3. Now that I think about it it would also be good to mention that a less updated version of the same is used. So that the user does not get so confused to see that the way to use Phaser is different than the one shown in the tutorial.

Comment (Graywolf9) [reply files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md:20]: I see, so please [suggest the change](https://github.com/mdn/content/edit/main/files/en-us/games/tutorials/html5_gamedev_phaser_device_orientation/index.md) on the [`en-US`](https://developer.mozilla.org/en-US/docs/Games/Tutorials/HTML5_Gamedev_Phaser_Device_Orientation) version, once approved we can merge this.

Thank you so much !

Comment (Graywolf9) [reply files/es/games/tutorials/html5_gamedev_phaser_device_orientation/index.md:20]: Hello ! @dabl03 I hope your having a great day, as this have already two weeks without action, I'm goint to merge this without the version number, once you suggest the change in the `en-US` article we can update this with that change.

Thank you so much for all your help

**Review by Graywolf9 — APPROVED** [review]
Thank you so much!

**Pull request merged**
